website/snippets/_render-method.md

```diff
@@ -0,0 +1,27 @@
+#### SQL compilation error when running the `--empty` flag on a model
```

Comment:
i think we can remove all this and have a simple sentence or two explaining that the `.render()` method allows users to explicitly call the ref/source relation. 

the `--empty` flag basically simulates running models without actually running them against your warehouse. its useful for testing models and checking for errors. 

so when the flag is used, it automatically applies filtering to `refs())` or  `sources()` -- meaning it may skip or modify how these references are handled for optimization. But there are cases (like in the [github issue](https://github.com/dbt-labs/dbt-core/issues/10673) where you want dbt to full process the ref or source (in the user's case, it was the source). so in order to run that, you can use the `.render()` method to explicitly tell dbt to compile and interpret the ref/source rather than have it automatically filtered.